deploy/submit.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#
+# Markus Dolensky, ICRAR, 2020-10-01
+#
+# Usage: sbatch leapjob.sh [OPTIONS]
+#  Options:
+#  -h,--help                   Print this help message and exit
+#  -s,--stations INT           Override number of stations to use in the measurement set
+#  -f,--filepath TEXT          MeasurementSet file path
+#  -d,--directions TEXT        Direction calibrations
+#  -i,--implementation TEXT    Compute implementation type
+#  -c,--config TEXT            Config filepath
+#
+# examples:
+#  sbatch ./bin/leapjob.sh --help
+#  sbatch ./bin/leapjob.sh -f ~/leap-accelerate/testdata/1197638568-32.ms -s 126 -i cuda -d 
```

Comment:
this data is in an absolution path, is it possible to make it a relative path?

Reply:
@calgray, it's a usage example. The value of option -f is passed to the script and that value can be relative or absolute.